src/domains/sign/io_sign_issuer_func.tf

```diff
@@ -0,0 +1,57 @@
+locals {
+  issuer_func_name      = format("%s-issuer-func", local.project)
+  issuer_func_container = "src"
+  issuer_func_package   = format("https://%sst.blob.core.windows.net/%s/%s.zip", replace(local.issuer_func_name, "-", ""), local.issuer_func_container, var.io_sign_issuer_func.version)
+}
+
+module "io_sign_issuer_func" {
+  source    = "git::https://github.com/pagopa/azurerm.git//function_app?ref=v2.18.2"
+  name      = local.issuer_func_name
+  subnet_id = module.io_sign_snet.id
+
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+  app_service_plan_info = {
+    kind                         = "Linux"
+    sku_tier                     = var.io_sign_issuer_func.sku_tier
+    sku_size                     = var.io_sign_issuer_func.sku_size
+    maximum_elastic_worker_count = 1
+  }
+
+  os_type           = "linux"
+  always_on         = true
+  linux_fx_version  = "NODE|16"
+  health_check_path = "api/v1/sign/info"
+
+  app_settings = {
+    FUNCTIONS_WORKER_RUNTIME         = "node"
+    WEBSITE_NODE_DEFAULT_VERSION     = "16.13.0"
+    WEBSITE_VNET_ROUTE_ALL           = "1"
+    WEBSITE_RUN_FROM_PACKAGE         = local.issuer_func_package
+    AzureWebJobsDisableHomepage      = "true"
+    CosmosDbConnectionString         = module.key_vault_secrets.values["CosmosDbConnectionString"].value
+    CosmosDbDatabaseName             = module.cosmosdb_sql_database_issuer.name
+    StorageAccountConnectionString   = module.key_vault_secrets.values["StorageAccountConnectionString"].value
+    IssuerUploadedBlobContainerName  = azurerm_storage_container.uploaded_documents.name
+    IssuerValidatedBlobContainerName = azurerm_storage_container.validated_documents.name
+    WaitingMessageQueueName          = azurerm_storage_queue.waiting_message.name
+    IOApiBasePath                    = ""
+    IOApiSubscriptionKey             = module.key_vault_secrets.values["IOApiSubscriptionKey"].value
+    IOApiRequestTimeout              = 10000
+    TokenizerApiBasePath             = ""
+    TokenizerApiSubscriptionKey      = module.key_vault_secrets.values["TokenizerApiSubscriptionKey"].value
```

Comment:
new names for tokenizer variables = `PdvTokenizerApiBasePath`, `PdvTokenizerApiKey`

Reply:
no default